### ISSUE TITLE
Support multiple inputs (tx) in transfer

### ISSUE THREAD

**Issue by sbellem** [bigchaindb/bigchaindb-driver#113]
As an example, we wish to support the now outdated use case outlined in https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/python-server-api-examples.html#multiple-inputs-and-outputs

``` python
# Create some assets for bulk transfer
for i in range(3):
    tx_mimo_asset = b.create_transaction(b.me, testuser1_pub, None, 'CREATE')
    tx_mimo_asset_signed = b.sign_transaction(tx_mimo_asset, b.me_private)
    b.write_transaction(tx_mimo_asset_signed)

# Wait until they appear on the bigchain and get the inputs
owned_mimo_inputs = b.get_owned_ids(testuser1_pub)

# Check the number of assets
print(len(owned_mimo_inputs))

# Create a signed TRANSFER transaction with all the assets
tx_mimo = b.create_transaction(testuser1_pub, testuser2_pub, owned_mimo_inputs, 'TRANSFER')
tx_mimo_signed = b.sign_transaction(tx_mimo, testuser1_priv)

# Write the transaction
b.write_transaction(tx_mimo_signed)

# Check if the transaction is already in the bigchain
tx_mimo_retrieved = b.get_transaction(tx_mimo_signed['id'])
```

**Comment by TimDaub** [comment]
[bigchaindb/bigchaindb-common#56](https://github.com/bigchaindb/bigchaindb-common/issues/56) related/blocks this ticket.

**Comment by sbellem** [comment]
Has been resolved by #109, and `.create` and `.transfer` are deprecated: #149 .
